### ISSUE TITLE
Laravel 11 support

### ISSUE THREAD

**Issue by przwiers** [butschster/kraken-api-client#24]
Is it possible to make this package compatible with Laravel 11?

Would be great, thanks!

Paul

**Comment by butschster** [comment]
Done

**Comment by przwiers** [comment]
Thanks for the quick response, but it seems that there are some more dependencies that need to be changed?


``` Problem 1
    - butschster/kraken-api-client v2.8.0 requires brick/money ^0.8.0 -> satisfiable by brick/money[0.8.0, 0.8.1].
    - butschster/kraken-api-client[v1.0, ..., v1.3] require guzzlehttp/guzzle ^6.3 -> found guzzlehttp/guzzle[6.3.0, ..., 6.5.8] but the package is fixed to 7.8.1 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - butschster/kraken-api-client[v1.4, ..., v1.5] require nesbot/carbon ^2.16 -> found nesbot/carbon[2.16.0, ..., 2.72.3] but the package is fixed to 3.3.1 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - butschster/kraken-api-client[v2.0, ..., v2.3] require illuminate/support ^8 -> found illuminate/support[v8.0.0, ..., v8.83.27] but these were not loaded, likely because it conflicts with another require.
    - butschster/kraken-api-client[v2.4, ..., v2.5] require illuminate/support ^8|^9.0 -> found illuminate/support[v8.0.0, ..., v8.83.27, v9.0.0, ..., v9.52.16] but these were not loaded, likely because it conflicts with another require.
    - butschster/kraken-api-client[v2.6, ..., v2.7.1] require illuminate/support ^8|^9.0|^10.0 -> found illuminate/support[v8.0.0, ..., v8.83.27, v9.0.0, ..., v9.52.16, v10.0.0, ..., v10.48.10] but these were not loaded, likely because it conflicts with another require.
    - brick/money[0.8.0, ..., 0.8.1] require brick/math ~0.10.1 || ~0.11.0 -> found brick/math[0.10.1, 0.10.2, 0.11.0] but the package is fixed to 0.12.1 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires butschster/kraken-api-client * -> satisfiable by butschster/kraken-api-client[v1.0, ..., v1.5, v2.0, ..., v2.8.0].

```

**Comment by butschster** [comment]
Thx for response. I'll fix it soon 😵‍💫
